src/sentry/models/artifactbundle.py

```diff
@@ -0,0 +1,89 @@
+from enum import Enum
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+)
+
+
+class SourceFileType(Enum):
+    SOURCE = 1
+    MINIFIED_SOURCE = 2
+    SOURCE_MAP = 3
+    INDEXED_RAM_BUNDLE = 4
+
+    @classmethod
+    def choices(cls):
+        return [(key.value, key.name) for key in cls]
+
+
+@region_silo_only_model
+class ArtifactBundle(Model):
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField(db_index=True)
+    bundle_id = models.UUIDField(null=True)
+    file = FlexibleForeignKey("sentry.File")
+    artifact_count = BoundedPositiveIntegerField()
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_artifactbundle"
+
+        unique_together = (("organization_id", "bundle_id"),)
+
+
+@region_silo_only_model
+class ReleaseArtifactBundle(Model):
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField(db_index=True)
+    release_name = models.CharField(max_length=250)
```

Comment:
Should this be `release_id` and related to the `Release` table?

Reply:
@mitsuhiko wanted a weak reference to releases and decided to use the `release.version`.